etcd/os-faq.md

```diff
@@ -0,0 +1,48 @@
+# etcd on Container Linux FAQ
+
+Are you planning on running etcd on CoreOS Container Linux? The following FAQ clarifies some common misconceptions.
+
+## What is the current version of etcd?
+
+[v3][etcd-latest-version].
+
+## Where do I find etcd v3 in Container Linux?
+
+Container Linux includes a systemd service, [etcd-member.service][etcd-member-service], which pulls and runs a desired version of etcd (defaults to v3) in a rkt container. This functions as if the etcd binary was being run, but does not require the binary to be installed on your Container Linux node.
+
+Newer versions of the etcd binary (v3) are not planned to be included in the OS.
+
+## Why is etcd v2 still in the os?
+
+The etcd binary will be included in the OS until June, 2018.
+
+The preferred way to run etcd in Container Linux is to use the etcd-member service mentioned above in the first question.
+
+## When will v3 be included in the OS image?
+
+Never.
```

Comment:
Thank you. I'm glad you noticed. :D
We'll give this a pass for external palatability before any merge.